examples/solenoid/run_solenoid.py

```diff
@@ -20,7 +20,7 @@
 # domain decomposition & space charge mesh
 sim.init_grids()
 
-# load a 2 GeV proton beam with an initial
+# load a 250 MeV proton beam with an initial
 # normalized transverse rms emittance of 1 um
```

Comment:
The documentation should read:

# load a 250 MeV proton beam with an initial horizontal rms emittance
# of 1 um and an initial vertical rms emittance of 2 um

Reply:
Thanks a lot, fixed in both scripts.